ingestors/npm.go

```diff
@@ -39,16 +38,11 @@ func (ingestor *NPM) Name() string {
 }
 
 func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
-	since, err := getBookmark(ingestor, nil)
```

Comment:
Can you at least split into a separate commit? :)

Reply:
Yes, haha. I'll break out another PR.